test/performance/specs/post-editor.spec.js

```diff
@@ -418,9 +418,11 @@ test.describe( 'Post Editor Performance', () => {
 			const globalInserterToggle = page.getByRole( 'button', {
 				name: 'Toggle block inserter',
 			} );
-
 			// Open Inserter.
 			await globalInserterToggle.click();
+
+			await page.getByPlaceholder( 'Search' ).click();
```

Comment:
If you can scope this down to the specific piece of UI you're targetting it may help avoid future regressions.